### PR TITLE
ascii dag navigation

### LINKED ISSUE #503: Inform user how to exit pipeline ascii display
Context:
https://discordapp.com/channels/485586884165107732/563406153334128681/603890677176336394

https://dvc.org/doc/commands-reference/pipeline-show#options should probably contain note that `q` can be used to exit from `dvc pipeline show --ascii`

### PR DESCRIPTION
**Pull request by ryokugyu**
Fix #503

### CODE REVIEW

File: static/docs/commands-reference/pipeline_show.md [hunk 1]
```diff
@@ -116,6 +116,10 @@ instead of stages.
                 `--------------'
   ```
 
+  To navigate, use `W`, `A`, `S`, `D` keys.
```
Comment (efiop) [inline static/docs/commands-reference/pipeline_show.md]:
```suggestion
  To navigate, use arrows or `W`, `A`, `S`, `D` keys.
```

File: static/docs/commands-reference/pipeline_show.md [hunk 2]
```diff
@@ -116,7 +116,7 @@ instead of stages.
                 `--------------'
   ```
 
-  To navigate, use `W`, `A`, `S`, `D` keys.
+  To navigate, use arrows or `W`, `A`, `S`, `D` keys.
```
Comment (shcheklein) [inline static/docs/commands-reference/pipeline_show.md]: this note should be also part of the description and part of the `ascii` option above.

File: static/docs/commands-reference/pipeline_show.md [hunk 3]
```diff
@@ -116,7 +116,7 @@ instead of stages.
...
 
   To exit, press `Q`.
```
Comment (shcheklein) [inline static/docs/commands-reference/pipeline_show.md]: Make it part of the previous paragraph. No need to split it. Use `>` to make it a note.

File: static/docs/commands-reference/pipeline_show.md (hunk 2 above)
Comment (ryokugyu) [reply static/docs/commands-reference/pipeline_show.md]: after adding it to the `ascii` option.

pipeline show

**Pull request merged**